compiler/ballerina-lang/src/main/java/io/ballerina/projects/internal/model/Central.java

```diff
@@ -46,6 +55,42 @@ public String getAccessToken() {
         return accesstoken;
     }
 
+    /**
+     * Get the connection timeout.
+     *
+     * @return connection timeout
+     */
+    public int getConnectTimeout() {
```

Comment:
we use the convention of avoiding `get` in the method name of getters for public APIs.